package.json

```diff
@@ -26,5 +27,8 @@
     "hooks": {
       "pre-commit": "lint-staged"
     }
+  },
+  "devDependencies": {
+    "@types/node": "^14.14.31"

```

Comment:
To tak samo

Reply:
A no tak sorka, nie ogarnąłem, że jest oddzielnie package.json :) wieczorem poprawię